gnd/src/main/java/com/google/android/gnd/ui/home/mapcontainer/MapContainerViewModel.java

```diff
@@ -90,6 +98,11 @@
                 .map(Loadable::value)
                 .switchMap(this::getFeaturesStream)
                 .map(MapContainerViewModel::toMapPins));
+    this.mbtiles =
```

Comment:
mbtilesFilePaths maybe? (since this doesn't actually contain the data)

Reply:
f9a41a2650397ca3f118ba6677bb57e2affb43af